### DISCUSSION TITLE
VSCode intellisense for .env.local

### DISCUSSION THREAD

**Discussion by pandydavey** [vercel/next.js#16835] [Help]
Not sure if someone is able to help me figure out how to solve this issue, but i have a .env.local file that i use locally (the clue is in the name) i want intellisense to be able to pick up the values from that file when i type `process.env.`. Currently VSCode only shows NODE_ENV. It's not a major drama having the .env.local file open somewhere and refer to it, but i would prefer to be able to have the IDE do that part of the legwork. 

Is there an extension? or a setting that i can use? There must be a way to do this. 

Thanks in advance.

**Comment by jmevel** [comment]
Three years later, it seems nothing has changed. I just created a [Github issue](https://github.com/dotenv-org/dotenv-vscode/issues/90)
